Changelog.md

```diff
@@ -1,5 +1,39 @@
 # Changelog
 
+### 1.1.3
+
+**Image:**
+- k8s.gcr.io/ingress-nginx/controller:v1.1.3@sha256:31f47c1e202b39fadecf822a9b76370bd4baed199a005b3e7d4d1455f4fd3fe2
+
+This release bumps Alpine to 3.14.4 and nginx to 1.19.10 
```

Comment:
We need to get explicit that we patched for openssl CVE-2022-0778 in this description